lib/timer.js

```diff
@@ -201,26 +201,33 @@ timers.create(id, options, (err) => {
       debug('setting timer', id);
       //TODO(esatterwhite):
       // what should happen if leveldb fails.
-      if (err) console.error(err);
+      if (err) {
+        console.error(err);
+        cb(err, null);
+      }
       this.nats.publish('skyring:events', JSON.stringify({
         type: EVENT_STATUS.CREATED
       , timer: id
       , node: this[kNode]
       , created: data.created
       , payload: payload
-      }), noop);
-
-      data.timer = setTimeout(
-        transport
-      , payload.timeout - elapsed
-      , payload.callback.method
-      , payload.callback.uri
-      , payload.data
-      , id
-      , this
-      ).unref();
-      this.set( id, data );
-      cb(null, id);
+      }), (err) => {
+        if (err) {
+          console.error(err);
+          cb(err, null);
```

Comment:
There isn't really a need to capture an error on this. it is for monitoring. 
this specific call failing shouldn't effect anything else.

lines 208-214 could  be removed and this should behave the same.